meson.build

```diff
@@ -81,6 +81,7 @@ if get_option('disable-dupe-detection')
   add_global_arguments(['--define=NO_DUPE_DETECTION'], language : vala)
 endif
 
+face_resources = (['main.vala']) #hack to avoid error when no face option used
```

Comment:
Should be possible to use just []. But please move the definition of the faces things into src/meson.build